src/superqt/collapsible/_collapsible.py

```diff
@@ -0,0 +1,161 @@
+"""A collapsible widget to hide and unhide child widgets"""
+import pathlib
+from typing import Union
+
+from PySide2.QtCore import QPropertyAnimation, QVariantAnimation
+from PySide2.QtGui import QTransform
```

Comment:
use `superqt.qtcompat` instead of `PySide2` so we can also support PyQt

Reply:
ahh... vscode auto-import strikes again :( I have been fighting it but it seems i missed a few.